src/utils/assets.js

```diff
@@ -30,6 +30,20 @@ import get from 'lodash.get';
 import { ETH } from 'constants/assetsConstants';
 import { formatAmount, isCaseInsensitiveMatch } from 'utils/common';
 
+const sortAssetsFn = (a: Asset, b: Asset): number => {
+  return a.symbol.localeCompare(b.symbol);
+};
+
+export const sortAssetsArray = (assets: Asset[]): Asset[] => {
+  return assets.sort(sortAssetsFn);
+};
+
+export const sortAssets = (assets: Assets): Asset[] => {
+  const assetsList = Object.keys(assets).map((key: string) => assets[key]);
```

Comment:
I think you can use `getAssetsAsList` here

Reply:
Didn't realise that. Updated.

Reply:
no worries, it's a new thing ;)